### PR TITLE
Improving scheduler queue unit test coverage

### PR DESCRIPTION
**Pull request by sathyanarays**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Improves test coverage of scheduler queue

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by alculquicondor** [comment]
/unassign

Please let a reviewer take a look first.

File: pkg/scheduler/internal/queue/scheduling_queue_test.go [hunk 1]
```diff
@@ -350,6 +350,19 @@ func TestPriorityQueue_Update(t *testing.T) {
...
+	q.AddUnschedulableIfNotPresent(q.newQueuedPodInfo(medPriorityPodInfo.Pod), q.SchedulingCycle())
+	if len(q.unschedulablePods.podInfoMap) != 1 {
+		t.Error("Expected unschedulablePods to be 1.")
+	}
+
+	updatedPod = medPriorityPodInfo.Pod.DeepCopy()
+	q.Update(medPriorityPodInfo.Pod, updatedPod)
+	if len(q.unschedulablePods.podInfoMap) != 1 {
+		t.Error("Expected unschedulablePods to be 1.")
+	}
```
[the remark this answers is not in the document]
Comment (sathyanarays) [reply pkg/scheduler/internal/queue/scheduling_queue_test.go]: Done

File: pkg/scheduler/internal/queue/scheduling_queue_test.go [hunk 2]
```diff
@@ -342,6 +342,12 @@ func TestPriorityQueue_Update(t *testing.T) {
 		t.Error("Expected unschedulablePods to be 1.")
 	}
 	updatedPod = medPriorityPodInfo.Pod.DeepCopy()
+
+	q.Update(medPriorityPodInfo.Pod, updatedPod)
```
Comment (alculquicondor) [inline pkg/scheduler/internal/queue/scheduling_queue_test.go:346]: what are we testing here? can you add a comment?

Comment (alculquicondor) [reply pkg/scheduler/internal/queue/scheduling_queue_test.go:346]: or better, use `t.Log` to indicate what conditions changed.
If you can, add similar notes for the existing tests.

Arguably, these tests are very old and need some love.

**Pull request closed without merging**